Update variable-sets.mdx (#2241)
hashicorp/terraform-website: content/cloud-docs/api-docs/variable-sets.mdx

A [variable set](/cloud-docs/workspaces/variables#scope) is a resource that allows you to reuse the same variables across multiple workspaces. For example, you could define a variable set of provider credentials and automatically apply it to one or all workspaces.

> You need [`read variables` permission](/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions) to view the variables for a particular workspace and to view the variable sets in the owning organization. In order to create or edit variable sets a team, the user account is part of, needs to have at least `Manage Workspace` organizational access. A team's organizational access can be managed via the [Team Management](https://www.terraform.io/cloud-docs/users-teams-organizations/teams#managing-teams) settings page of the organization.

Reviewer: ```suggestion
You need [`read variables` permission](/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions) to view the variables for a particular workspace and to view the variable sets in the owning organization. To create or edit variable sets, your team must have [`Manage Workspace` organization access](/cloud-docs/users-teams-organizations/teams#managing-organization-access). 
```
